base/abstractarray.jl

```diff
@@ -687,6 +688,10 @@ julia> empty([1.0, 2.0, 3.0], String)
 ```
 """
 empty(a::AbstractVector{T}, ::Type{U}=T) where {T,U} = Vector{U}()
```

Comment:
change this method also then?

Reply:
I left that in for inlining purposes due to all the existing code depending on it. If that is not necessary, then I'll change it as well.